README.md

```diff
@@ -142,10 +177,12 @@ and only pass the role name in the `iam.amazonaws.com/role` annotation, otherwis
 ```
 $ kube2iam --help
 Usage of kube2iam:
+      --add-rule                       Add iptables rule (requires HOST_IP environment variable)
```

Comment:
I'd rather call this flag `--iptables` or `--add-iptables-rule` if we want to be clearer.